### ISSUE TITLE
`background` property on the table contains invalid value

### ISSUE THREAD

**Issue by mastertheblaster** [mjmlio/mjml#1367]
The following mjml:

```html
<mjml>
  <mj-body>
    <mj-section>
      <mj-column padding="10px" background-color="red">
        <mj-divider border-color="#F45E43"></mj-divider>
      </mj-column>
    </mj-section>
  </mj-body>
</mjml>
```

creates HTML with a `table`, having property `background` which is equal to the specified color.
In this case `<table background="red"/>`. `background` property is used to specify the image, not the color. Color should be specified using the `bgcolor` property.

Is this a workaround for some issue or just by accident?

**Comment by iRyusa** [comment]
Hi,

Yeah this should be removed and the `container-background-color` one too.

**Comment by iRyusa** [comment]
It seems that it has been fixed in recent 4.2 release, so i'm closing
